src/Evolution/Systems/CurvedScalarWave/BoundaryConditions/RegisterDerived.cpp

```diff
@@ -0,0 +1,15 @@
+// Distributed under the MIT License.
```

Comment:
If I understand the new registration mechanism correctly these files are no longer needed.